examples/walkthrough/k8s201.md

```diff
@@ -48,15 +95,40 @@ spec:
             - containerPort: 80
 ```
 
-### Services
+#### Replication Controller Management
+
+If you have the source checked out or downloaded the pre-compiled release you should have an `examples` directory in the Kubernetes root directory. If not, download the [nginx replication controller example definition](replication-controller.yaml).
```

Comment:
This sentence does not appear above for the labelled pod. There it's assumes the example directory exists.